src/pages/modals/contact-modals/contact-modal.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from "@angular/core";
+import { NavParams, ViewController } from "ionic-angular";
+
+@Component({
+    selector: "page-contact-modal",
+    templateUrl: "contact-modal.html",
+})
+export class ContactModalPage {
+
+    public contact;
+
+    constructor(public viewCtrl: ViewController,
+                public params: NavParams) {
+        this.contact = this.params.get('contact');
+    }
+
+    dismiss() {
+        this.viewCtrl.dismiss();
+    }
```

Comment:
Do you need to catch this promise? I know Ionic sometimes throws errors with uncaught promises if you run it twice in a row. To fix it, just add dismiss().catch(f => f );

Also, make dismiss either public or private.